crates/iceberg-catalog/src/service/authz/mod.rs

```diff
@@ -422,8 +422,9 @@ where
         {
             Ok(())
         } else {
+            let actor = metadata.actor();
             Err(ErrorModel::forbidden(
-                format!("Forbidden action {action} on project {project_id}"),
+                format!("Forbidden action {action} on project {project_id} for actor {actor:?}"),
```

Comment:
Let's implement display for actor, debug formats can look a bit funky on public apis

Reply:
Done